app/views/ncr/work_orders/form.html.erb

```diff
@@ -72,10 +72,13 @@
     </div>
     <div class="form-group">
       <%= field_set_tag "Amount", class: "required" do %>
-        <%= f.label :amount, "Dollar Amount", :class => 'sr-only'  %>
+        <%= f.label :amount, "Dollar Amount", class: 'sr-only' %>
```

Comment:
Some people like fancy cars and diamond rings...just get @jessieay Ruby 1.9 Hash syntax.